.github/workflows/grumpy-reviewer.lock.yml

```
@@ -0,0 +1,1609 @@
+      - safe_outputs
+    if: >
+      always() && (needs.agent.result != 'skipped' || needs.activation.outputs.lockdown_check_failed == 'true' ||
+      needs.activation.outputs.stale_lock_file_failed == 'true' || needs.activation.outputs.daily_ai_credits_exceeded == 'true')
+    runs-on: ubuntu-slim
+    permissions:
+      contents: read
+      pull-requests: write
+    concurrency:
+      group: "gh-aw-conclusion-grumpy-reviewer"
+      cancel-in-progress: false
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)